### PR TITLE
good looking

### PR DESCRIPTION
**Pull request by bejelentkezni**
Several misc fixes, typos, branding consistency, and a tooltip clarification.

### CODE REVIEW

File: docs/save_frame_placeholders.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 | Placeholder  | Description                          | Example   |
 | ------------ | ------------------------------------ | --------- |  
 | script_name  | Script name without extension        | haruhi_01 |
-| index        | Video node number (zero-based)       | 0         |
+| index        | Video node number (zero-indexed)     | 0         |
```
Comment (Setsugennoao) [inline docs/save_frame_placeholders.md:6]: my brother in christ, lookup before changing stuff, it's zero-based counting system

File: docs/shortcuts.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
 | S                  | Toggle whether output nodes are synced                 |
 | V                  | Copy current frame number to clipboard                 |
 | Ctrl + R           | Reload script                                          |
+| Ctrl + Space       | Press/Toggle the selected button/checkbox              |
```
Comment (Setsugennoao) [inline docs/shortcuts.md:21]: Should be in the settings dialog category

File: docs/vim_install.md [hunk 3]
```diff
@@ -16,14 +16,14 @@ The following is a way to integrate VS-Preview into Vim.
     1. Browse to Vim and click `Open`.
         * To get correct syntax highlighting, add `au BufReadPost,BufNewFile *.vpy setlocal syntax=python` to your `_vimrc`.
 
-## Running scripts with VSPreview in Vim
+## Running scripts with vs-preview in Vim
 
-Now that you've configured your current project to use Vim. It's time to run it. 
+Now that you've configured your scripts to use Vim, it's time to run them.
```
Comment (Setsugennoao) [inline docs/vim_install.md:21]: all edits but the comma are just wrong

File: docs/vscode_install.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # Installation in VS Code
 
-Vapoursynth Preview can be used in a number of different ways.<br>One of them is directly in Microsoft's free code editor [Visual Studio Code](https://code.visualstudio.com).
+vs-preview can be used in a number of different ways.<br>One of them is directly in Microsoft's free code editor [VS Code](https://code.visualstudio.com).
```
Comment (Setsugennoao) [inline docs/vscode_install.md:3]: why did you feel the need to shorten visual studio as vs? that can be confused with vapoursynth

File: README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 Fork of Endilll's [vapoursynth-preview](https://github.com/Endilll/vapoursynth-preview) (not maintained anymore)
 
-This program is meant to be paired with a code editor with integrated terminal like Visual Studio Code.
+This program is meant to be paired with a code editor with integrated terminal like VS Code or Vim.
```
Comment (Setsugennoao) [inline README.md:5]: why did you feel the need to shorten visual studio as vs? that can be confused with vapoursynth

File: docs/vscode_install.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
 
-The following is a way to integrate VS-Preview into VSCode. 
+The following is a way to integrate vs-preview into VS Code.
```
Comment (Setsugennoao) [inline docs/vscode_install.md:5]: change this to Visual Studio Code

File: docs/vscode_install.md [hunk 7]
```diff
@@ -16,12 +16,12 @@ The following is a way to integrate VS-Preview into VSCode.
     1. Select the "Add Item" button.
     1. Enter `*.vpy` for the "Item".
     1. Enter `python` for the "Value".
-    1. Select the "OK" button.
+    1. Select the "Okay" button.
```
Comment (Setsugennoao) [inline docs/vscode_install.md:19]: It's OK, not Okay
it_is_ok_not_okay

File: docs/vscode_install.md [hunk 8]
```diff
@@ -16,12 +16,12 @@ The following is a way to integrate VS-Preview into VSCode.
...
 
 ## Setting Up a `launch.json` File
 
 To be able to run the file you're currently working on,<br>you'll need to set up a launch.json file,
-which tells Visual Studio Code what to do to run it with VS Preview
+which tells VS Code what to do to run it with vs-preview
```
Comment (Setsugennoao) [inline docs/vscode_install.md:24]: yada yada, don't shorten Visual Studio

File: docs/vscode_install.md [hunk 9]
```diff
@@ -36,11 +36,11 @@ This is easier said then done
     ```
 1. Congratulations, you can now write and preview beautiful filterchains!
 
-**Note:** Step 3 will need to be completed for every new project unless you've saved your `launch.json` directly in `~/.vscode`.
+**Note:** Step 3 will need to be completed for every new project unless you've saved your `launch.json` directly in `~/.vscode`, or `%USERPROFILE/.vscode%` on Windows.
 
-## Running scripts with VSPreview in VSCode
+## Running scripts with vs-preview in VS Code
```
Comment (Setsugennoao) [inline docs/vscode_install.md:41]: expand VS Code into Visual Studio Code

**Review by Setsugennoao — CHANGES_REQUESTED** [review]

**Comment by bejelentkezni** [comment]
I'll just open it against Light's fork.

**Pull request closed without merging**